neo.UnitTests/UT_Consensus.cs

```diff
@@ -79,7 +79,7 @@ public void ConsensusService_Primary_Sends_PrepareRequest_After_OnStart()
 
             Console.WriteLine($"header {header} hash {header.Hash} timstamp {timestampVal}");
 
-            timestampVal.Should().Be(4244941696); //1968-06-01 00:00:00
+            timestampVal.Should().Be(1514007552); //1968-06-01 00:00:00
```

Comment:
@igormcoelho, can you check this adjusted on these tests?

Reply:
This doesn't look right... it should increase, never decrease :joy: probably, it is overflowing some int32 size (should be int64 or long, at least).

Reply:
kkkkkkkkkkk
I knew that....
Things need all be long for us. Ram never was a problem when things have vallgrind.

Reply:
What is this date?
How did you guys manage to make 4244941696 be 1968-06-01 00:00:00? Seems weird to me. Maybe the tests were broken all this time?